apps/website/src/pages/docs/inputwithdecorations.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Input with decorations
+description: Input component with various additional decorations
+layout: ./_layout.astro
+propsPath: '@itwin/itwinui-react/esm/core/utils/components/InputFlexContainer.d.ts'
+thumbnail: #TODO
+group: utilities
```

Comment:
Should these go under inputs instead of utilities?

Reply:
Updated